core-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/extension/elytron/jaas-realm/main/module.xml

```diff
@@ -33,10 +33,6 @@
         <exclude path="org/wildfly/extension/elytron/JaasCustomSecurityRealmWrapper"/>
     </exports>
```

Comment:
This export was not excluding `JaasCustomSecurityRealmWrapper` as seems to be the intention, exports/exclude works only with paths and not with specific class resources, but JaasCustomSecurityRealmWrapper is not under a specific package so we cannot easily exclude it from this module export. We can safely remove it from here.